### PR TITLE
Added documentation for the ARVR server classes

### PR DESCRIPTION
**Pull request by BastiaanOlij**
Just adding a bunch of documentation about the new ARVR classes.

### CODE REVIEW

File: doc/base/classes.xml [hunk 1]
```diff
@@ -2007,32 +2007,49 @@
 </class>
 <class name="ARVRAnchor" inherits="Spatial" category="Core">
 	<brief_description>
+		Anchor point in AR Space
 	</brief_description>
 	<description>
+		The ARVR Anchor point is a spatial node that maps a real world location identified by the AR platform to
```
Comment (Noshyaar) [inline doc/base/classes.xml]: One line should be one paragraph if I'm not mistaken.

Comment (BastiaanOlij) [reply doc/base/classes.xml]: Ah!

**Comment by BastiaanOlij** [comment]
@Noshyaar , where the last changes I did sufficient or did I miss some?

**Comment by Noshyaar** [comment]
Seems fine, except I have seen somewhere that you shouldn't put blank lines between paragraphs (each line).

https://github.com/GDquest/godot-docs/blob/9cb30a854c784e1426e8ffcc3f20c1f7e3a3bfb4/community/contributing/updating_the_class_reference.rst

> Write in a clear and simple language. Always follow the :ref:`class_reference_styleguide` to keep your descriptions short and easy to read. Do not leave empty lines in the descriptions: each line in the XML file will result in a new paragraph.

**Comment by BastiaanOlij** [comment]
All new to me :) I'll remove the empty lines.

I guess some of the info might be better put in a tutorial or something so I guess I could shorten some of the things, though its also all pretty new stuff....

File: doc/base/classes.xml [hunk 2]
```diff
@@ -2007,32 +2007,42 @@
 </class>
 <class name="ARVRAnchor" inherits="Spatial" category="Core">
 	<brief_description>
+		Anchor point in AR Space
 	</brief_description>
 	<description>
+		The ARVR Anchor point is a spatial node that maps a real world location identified by the AR platform to a position within the game world. For example, as long as plane detection in ARKit is on, ARKit will identify and update the position of planes (tables, floors, etc) and create anchors for them.
+
```
Comment (akien-mga) [inline doc/base/classes.xml]: Each line in the XML is a paragraph, so empty newlines mean empty paragraphs in the docs. Could you remove those?

File: doc/base/classes.xml [hunk 3]
```diff
@@ -2091,12 +2115,14 @@
 			<argument index="0" name="axis" type="int">
 			</argument>
 			<description>
+				Returns the value of the given axis for things like triggers, touchpads, etc. that are embedded into the controller
```
Comment (akien-mga) [inline doc/base/classes.xml]: Many descriptions lack a dot at the end of the sentence.

Comment (BastiaanOlij) [reply doc/base/classes.xml]: Yes punctuation isn't my strong point ;) let me have a look at those too

**Comment by BastiaanOlij** [comment]
Ok, blank lines have been removed and I hope I found all the punctuation issues :)

**Pull request merged**